### DISCUSSION TITLE
content-length

### DISCUSSION THREAD

**Discussion by anjanesh** [mpdf/mpdf#1744] [Q&A] [answered]
Is there a way to get content-length header before generating the PDF ?

```
$mpdf->WriteHTML($html);

# Set header
# header('Content-Length:' . filesize($file)); # How do I do this ?

$mpdf->Output('MyFileName.pdf', 'I');

```

**Comment by finwe** [comment, marked as answer]
You don't, the content of the file is generated when the Output is called.

You can output the content to a string variable, get the length of the string, set the inline pdf content headers and send the content yourself
